src/assets/data/medicines.json

```diff
@@ -1,144 +1,144 @@
 [
-  "Accupril",
-  "Acebutolol",
-  "Aceon",
-  "Adalat CC",
-  "Aldactazide",
-  "Aldactone",
-  "Aldomet",
-  "Alpha methyldopa",
-  "Altace",
-  "Amiloride hydrochloride",
-  "Amlodipine besylate",
-  "Apresoline",
-  "Atacand",
-  "Atenolol",
-  "Avapro",
-  "Benazepril hydrochloride",
-  "Bepridil",
-  "Betapace",
-  "Betaxolol",
-  "Bisoprolol fumarate",
-  "Blocadren",
-  "Bumetanide",
-  "Bumex",
-  "Calan SR",
-  "Candesartan",
-  "Capoten",
-  "Captopril",
-  "Cardene SR",
-  "Cardizem CD",
-  "Cardizem SR",
-  "Cardura",
-  "Carteolol hydrochloride",
-  "Cartrol",
-  "Carvedilol",
-  "Catapres",
-  "Chlorothiazide",
-  "Chlorthalidone",
-  "Clonidine hydrochloride",
-  "Coreg",
-  "Corgard",
-  "Covera HS",
-  "Cozaar",
-  "Dilacor XR",
-  "Diltiazem hydrochloride",
-  "Diovan",
-  "Diuril",
-  "Doxazosin mesylate",
-  "Dyazide",
-  "DynaCirc",
-  "DynaCirc CR",
-  "Dyrenium",
-  "Enalapril maleate",
-  "Eprosartan mesylate",
-  "Esidrix",
-  "Felodipine",
-  "Fosinopril sodium",
-  "Furosemide",
-  "Guanabenz acetate",
-  "Guanadrel",
-  "Guanethidine monosulfate",
-  "Guanfacine hydrochloride",
-  "Hydralazine hydrochloride",
-  "Hydrochlorothiazide",
-  "Hydrodiuril",
-  "Hygroton",
-  "Hylorel",
-  "Hytrin",
-  "Indapamide",
-  "Inderal",
-  "Irbesarten",
-  "Ismelin",
-  "Isoptin SR",
-  "Isradipine",
-  "Kerlone",
-  "Labetalol hydrochloride",
-  "Lasix",
-  "Levatol",
-  "Lisinopril",
-  "Loniten",
-  "Lopressor",
-  "Losartan potassium",
-  "Lotensin",
-  "Lotrel",
-  "Lozol",
-  "Mavik",
-  "Maxzide",
-  "Methyldopa",
-  "Metolazone",
-  "Metoprolol succinate",
-  "Metoprolol tartrate",
-  "Micardis",
-  "Microzide",
-  "Midamar",
-  "Minipress",
-  "Minoxidil",
-  "Moduretic",
-  "Moexipril",
-  "Monopril",
-  "Mykrox",
-  "Nadolol",
-  "Nicardipine",
-  "Nifedipine",
-  "Nisoldipine",
-  "Normodyne",
-  "Norvasc",
-  "Penbutolol sulfate",
-  "Perindopril",
-  "Pindolol",
-  "Plendil",
-  "Prazosin hydrochloride",
-  "Prinivel",
-  "Procardia XL",
-  "Propranolol hydrochloride",
-  "Quinapril hydrochloride",
-  "Ramipril",
-  "Reserpine",
-  "Sectral",
-  "Serpasil",
-  "Solotol hydrochloride",
-  "Spironolactone",
-  "Sular",
-  "Telmisartan",
-  "Tenex",
-  "Tenormin",
-  "Terazosin hydrochloride",
-  "Teveten",
-  "Timolol maleate",
-  "Toprol-XL",
-  "Trandate",
-  "Trandolapril",
-  "Triamterene",
-  "Univasc",
-  "Valsartan",
-  "Vasocor",
-  "Vasotec",
-  "Verapamil hydrochloride",
-  "Visken",
-  "Wytensin",
-  "Zaroxolyn",
-  "Zebeta",
-  "Zestril",
-  "Ziac"
+  {"name": "Accupril"},
```

Comment:
This change was because this was failing on Android emulators for me, so I removed the map and made it use the map return value